.github/workflows/pod_lib_lint.yml

```diff
@@ -0,0 +1,33 @@
+name: pod lib lint
+
+on:
+  push:
+    branches: [master]
+    paths: 
+      - '.github/workflows/pod_lib_lint.yml'
+      - '*.podspec'
+      - 'Gemfile*'
+      - 'Yams.xcodeproj/**'
```

Comment:
Is this needed?

Reply:
Oh, I didn't know that cocoapods does not require `xcodeproj`.